feature-group-definitions/grid-animation.yml

```
@@ -0,0 +1,11 @@
+spec: https://drafts.csswg.org/css-grid-2/
```

Chrome (107) and Safari (16) only recently implemented animation for grid. If they were included in `grid`, it would  prevent baseline status for grid, which seemed wrong. I separated the animation features into their own group, shown in this file.

---

Is https://drafts.csswg.org/css-grid-2/ here instead of https://drafts.csswg.org/css-grid-3/ intentional? Since there's not specific section for it, using the same URL as the `grid` feature makes the most sense to me.

---

It really needs to be _this_:

```suggestion
spec: https://drafts.csswg.org/css-grid-2/#track-sizing
```

The level 3 spec doesn't address the animation of the properties (it doesn't cover the properties at all).